samples/nrf9160/lwm2m_client/prj.conf

```diff
@@ -76,17 +108,14 @@ CONFIG_SETTINGS_FCB=y
 # LTE link control
 CONFIG_LTE_LINK_CONTROL=y
 CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=y
```

Comment:
What is the reason for adding this? Just asking out of curiosity

Reply:
Isn't it necessary to enable this to have concurrent LTE and GPS?

Reply:
I'm asking because there is also the NB-IoT to be considered. Please update the NB-IoT overlay file _samples/nrf9160/lwm2m_client/overlay-nbiot.conf_ accordingly.

Reply:
I'm not certain does the prj.conf take precedence over the overlay conf still...

Reply:
Unless you use `DCONF_FILE` to specify configuration, the overlay config will overwrite the `prj.conf` configuration.

Reply:
The overlay should take precedence as far as I've experienced. Isn't that why it's called an overlay?

Reply:
It does, but it is a different setting. We could fix it by setting lte_m_gps=n and nbiot_gps=y in the overlay though

Reply:
Changed nbiot overlay as described